misc/ls-extensions/modules/trigger-service/src/main/java/io/ballerina/trigger/BallerinaTriggerService.java

```diff
@@ -59,7 +67,17 @@
 @JavaSPIService("org.ballerinalang.langserver.commons.service.spi.ExtendedLanguageServerService")
 @JsonSegment("ballerinaTrigger")
 public class BallerinaTriggerService implements ExtendedLanguageServerService {
+    public static final String BALLERINA = "ballerina";
+    public static final String BALLERINAX = "ballerinax";
     private LanguageClient languageClient;
+    private static final Map<String, String> IN_BUILT_TRIGGERS = Map.of(
+        "10006", "ftp",
```

Comment:
why did we decide to have this id as number rather than something we can compose using org and package name. If this id is going to stored in the client side I suppose better to use a composed id.

Reply:
If this id is coming from the central it is okay to keep it as it is.

Reply:
The id is kept as a number to make the model in align with the other triggers we are getting from central(where the id is a number).

The id for inbuilt triggers are not coming from central. Central does not support these inbuilt triggers. Thats why we are adding the responses for the inbuilt triggers manually

Reply:
got it.